### PR TITLE
HOV-29 | Adds Alert component 

### PR DESCRIPTION
**Pull request by saurabhsutar192**
Adds Alert Component

## Description
A customizable Alert component with type, variant support etc

## Motivation and Context
Alert Component Needed!

## How Has This Been Tested?
Local Build Success and manual testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: lib/src/components/Button/Button.stories.tsx [hunk 1]
```diff
@@ -9,7 +9,6 @@ export const Controls: Story<ButtonProps> = ({
   margin,
   isBlock,
   children,
-  ...nativeProps
```
Comment (pruthvi2103) [inline lib/src/components/Button/Button.stories.tsx:12]: why changes here?

**Review by pruthvi2103 — CHANGES_REQUESTED** [review]
just one small question

File: lib/src/components/Button/Button.stories.tsx (hunk 1 above)
Comment (saurabhsutar192) [reply lib/src/components/Button/Button.stories.tsx:12]: slacked you the details

**Review by pruthvi2103 — APPROVED** [review]
LGTM

**Comment by pruthvi2103** [comment]
@saurabhsutar192 can you please resolve the code smells pointed out by sonar cloud

**Pull request merged**